src/Illuminate/Cache/ApcStore.php

```diff
@@ -35,48 +36,56 @@ public function __construct(ApcWrapper $apc, $prefix = '')
     /**
      * Retrieve an item from the cache by key.
      *
-     * @param  string  $key
+     * @param  \BackedEnum|\UnitEnum|string  $key
```

Comment:
Btw, `BackedEnum` is a `UnitEnum`, so technically, `UnitEnum` would suffice

Reply:
Whilst technically true, the use of both in docblocks is consistent with similar behaviour elsewhere in the framework.